xtend-website/_config.yml

```diff
@@ -36,10 +36,10 @@ kramdown:
     coderay_css:               style
 
 src:
-  xtext: "https://github.com/eclipse/xtext/blob/v2.7.0"
+  xtext: "https://github.com/eclipse/xtext/blob/v2.11.0"
 
 javadoc:
-  xtext: "http://download.eclipse.org/modeling/tmf/xtext/javadoc/2.8"
+  xtext: "http://download.eclipse.org/modeling/tmf/xtext/javadoc/2.11"
```

Comment:
It was just a note.
We need a checklist for 2.11 release and add "Publish Javadoc 2.11" point to it.